server/game/cards/01_SOR/events/AttackPatternDelta.ts

```diff
@@ -0,0 +1,52 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+import { WildcardCardType } from '../../../core/Constants';
+
+export default class AttackPatternDelta extends EventCard {
+    protected override getImplementationId () {
+        return {
+            id: '1939951561',
+            internalName: 'attack-pattern-delta',
+        };
+    }
+
+    /**
+     * Build an immediate effect that modifies power and hp for the phase equal to the passed
+     * modification amount.
+     * @param modificationAmount The amount to increase power and hp by.
+     * @returns The effect.
+     */
+    private buildModifyStatsForPhaseImmediateEffect(modificationAmount: number) {
+        return AbilityHelper.immediateEffects.forThisPhaseCardEffect({
+            effect: AbilityHelper.ongoingEffects.modifyStats({ power: modificationAmount, hp: modificationAmount }),
+        });
+    }
+
+    public override setupCardAbilities () {
+        this.setEventAbility({
+            title: 'Give a friendly unit +3/+3 for this phase.',
+            targetResolver: {
```

Comment:
These need to be checking that it's a friendly unit (and the tests updated accordingly). Should just be `controller: RelativePlayer.Self`

Reply:
Done!